packages/design-system/src/components/slider/slider.css.ts

```diff
@@ -140,6 +140,22 @@ export const sliderClassNames: SliderClassNames = {
       '@layer': {
         [layers.components.l1]: {
           gridArea: 'track',
+          boxSizing: 'border-box',
+          zIndex: 'auto',
+          inlineSize: 'calc(100% - calc(calc(12px / 2)))',
+          minBlockSize: '12px',
+          verticalAlign: 'top',
+          marginInlineStart: 'calc(12px / 2)',
+          display: 'inline-block',
+          width: 'calc(100% - calc(calc(12px / 2) ) * 2)',
+          ':after': {
+            right: 'calc(calc(12px / 2) * -1)',
+            content: '',
+            inlineSize: 'calc(12px / 2)',
+            blockSize: '100%',
+            display: 'block',
+            position: 'absolute',
+          },
```

Comment:
should we decide to move forward with this implementation, all hardcoded sizes will be exchanged for variables